lib/active_record/virtual_attributes/virtual_total.rb

```diff
@@ -116,4 +116,4 @@ def virtual_aggregate_arel(reflection, method_name, column)
   end
 end
 
-ActiveRecord::Base.include VirtualAttributes::VirtualTotal
+ActiveSupport.on_load(:active_record) { include VirtualAttributes::VirtualTotal }
```

Comment:
Is the block in the scope of ActiveRecord::Base?  (How does this work?)

Reply:
ooh. good point. That is what Rubocop asked me to change, but I see what you mean.

Not totally sure how this works, I'm going to roll this back.